src/Modules/OrchardCore.Commerce/Assets/Scripts/stripe-card-form.js

```diff
@@ -36,40 +39,45 @@ window.stripeCardForm = function stripeCardForm(stripe, antiForgeryToken, urlPre
         payText.hidden = true;
     }
 
-    function enableInputs() {
-        formElements.forEach((element) => {
-            if (element.tagName === selectTagName) {
-                element.disabled = false;
-            }
+    function displayError(errors, container = errorContainer) {
+        if (!errors || errors.length === 0) {
+            container.hidden = true;
+            return;
+        }
 
-            element.readOnly = false;
-        });
-    }
+        const err = Array.isArray(errors) ? errors.filter((error) => error) : [errors];
 
-    function displayError(error) {
-        if (Object.prototype.hasOwnProperty.call(error, 'message')) {
-            errorContainer.textContent = error.message;
-        }
-        else {
-            errorContainer.textContent = error;
+        container.innerHTML = '<ul></ul>';
+        const ul = container.querySelector('ul');
+        for (let i = 0; i < err.length; i++) {
+            const li = document.createElement('li');
+            li.textContent = Object.prototype.hasOwnProperty.call(err[i], 'message') ? err[i].message : err[i];
+            ul.appendChild(li);
         }
 
-        enableInputs();
-
+        toggleInputs(true);
         card.update({ disabled: false });
-
         submitButton.disabled = false;
 
         paymentProcessingContainer.hidden = true;
         payText.hidden = false;
```

Comment:
I think `disableInputsAndShowSpinner()` could be "reworked" like `toggleInputs(enable)`, so instead of this code block, we could use it with a parameter here.